client/src/pages/BusinessDetails.js

```diff
@@ -1,89 +1,246 @@
+
+      // Map through the business' services array and compare the id string while mapping through the services whole data obj and using the find method to search for a matching id. If a match is found push the containing object to an array and return that array.
+      businessCopy?.services.map(bizService => {
+        // console.log(bizService)
+        dbServices?.services.find(service => {
+          if (service.id === bizService) {
+            tempBizServiceArr.push(service)
+          }
+        })
+      })
+
+      setBizServiceArr(tempBizServiceArr)
+      // console.log('bizServiceArr', bizServiceArr)
+    }
+    fetchDetails()
+    return function clean() {
+      setIsLoading(false)
+    }
+  }, [business])
+
   return (
-    <React.Fragment>
-      <ContentHeader title='Business Details ' />
-      <div className='card w-50 mx-auto'>
-        <div className='card-body'>
-          <ul className='products-list product-list-in-card'>
-            <li className='item'>
-              <figure className='indie-card'>
-                <div className='pic-star-container'>
-                  <img
-                    src='https://via.placeholder.com/100'
-                    alt='Placeholder Business Profile'
-                    className='image-fluid'
-                  />
-                  <p className='bus-details--rating '>
-                    <img
-                      src='star.svg'
-                      alt='Star Icon'
-                      style={{ width: '10%' }}
-                    />
-                    <img
-                      src='star.svg'
-                      alt='Star Icon'
-                      style={{ width: '10%' }}
-                    />
-                    <img
-                      src='star.svg'
-                      alt='Star Icon'
-                      style={{ width: '10%' }}
-                    />
-                    <img
-                      src='star.svg'
-                      alt='Star Icon'
-                      style={{ width: '10%' }}
-                    />
-                  </p>
-                  {props.token ? (
-                    <button className='btn btn-default'>Review</button>
-                  ) : (
-                    ''
-                  )}
-                </div>
-                <div>
-                  <h1 className='product title'>{business.businessName}</h1>
-                  <div>
-                    {business.address1} {business.city}, {business.state}{' '}
-                    {business.zipCode}
-                    <br />
-                    email: {business.email}
-                    <div className={phone ? 'visable-phone' : 'hidden-phone'}>
-                      phone: {business.phone}
+    <>
+      {!props.loading ? (
+        <>
+          {/* (Backlog) TODO: [ ] - ? Remove all ContentHeaders or modify and utilize across app   */}
+          {/* <ContentHeader title='Business Details ' /> */}
+          <section className='content'>
```

Comment:
i deleted content here because i have it wrapping around results in app.js.

Reply:
Hi @cynthiaeddy, to clarify in your submitted PRs you have deleted the line 99, <ContentHeader />, from your <BusinessDetails /> file?

Reply:
what i mean is that line 101 isn't necessary because the section content is already wrapping around results in app.js.